ckb_swappable_signatures.h

```diff
@@ -0,0 +1,55 @@
+#ifndef CKB_C_STDLIB_CKB_SWAPPABLE_SIGNATURES_H_
+#define CKB_C_STDLIB_CKB_SWAPPABLE_SIGNATURES_H_
+
+#include "ckb_dlfcn.h"
+
+#ifdef CKB_SWAPPABLE_SIGNATURES_DEBUG
+#define DEBUG(s) ckb_debug(s)
+#else
+#define DEBUG(s)
+#endif /* CKB_SWAPPABLE_SIGNATURES_DEBUG */
+
+typedef int (*verify_function)(void *, const uint8_t *, size_t, const uint8_t *,
+                               size_t, uint8_t *, size_t *);
+
+typedef struct {
+  uint8_t *code_buffer;
+  uint64_t code_buffer_size;
+  uint8_t *prefilled_data_buffer;
+  uint64_t prefilled_buffer_size;
+  verify_function verify_func;
+} CkbSwappableSignatureInstance;
```

Comment:
It is hard to get the meaning of `Swappable` here. Can we change the name to `CkbSecp256k1Context` or `CkbSecp256k1Instance`?

Reply:
It is named in collaboration with this [draft spec](https://talk.nervos.org/t/rfc-swappable-signature-verification-protocol-spec/4802), the code built here is more than just secp256k1